src/redux/modules/counter.js

```diff
@@ -1,17 +1,15 @@
-import createReducer from 'utils/createReducer'
+import { createAction,
```

Comment:
Not at all relevant to the PR, but just curious: is this a coding convention you use in your projects for imports? Never seen it before (not saying to change it, just curious). Looks kinda cool.